corehq/motech/templates/motech/log_detail.html

```diff
@@ -15,7 +15,13 @@
           {% if log.payload_id %}
           <tr>
             <th>{% trans "Payload ID" %}:</th>
-            <td>{{ log.payload_id }}</td>
+            <td>
+              {{ log.payload_id }} {% if is_dimagi %}
```

Comment:
It shouldn't be necessary to hide this for non-dimagi users. The search view will check domain access etc. so it is safe for normal users to use.

Reply:
Would we want the button to show up for non-dimagi users if they wouldn't be able to see any results when they click on it?

Reply:
From the JIRA ticket is seemed like it was non-Dimagi users who needed access, is that not correct? I think it would be fine to put the link there - if they don't have permissions to view the doc then they'll get a 403 page.

Reply:
Alternately we could build in a permission check into the `get_doc_info` function and then call that in the view and pass the URL to the template only if the user has permission.